Userland/Libraries/LibWeb/HTML/HTMLMediaElement.h

```diff
@@ -260,6 +261,9 @@ class HTMLMediaElement : public HTMLElement {
     // https://html.spec.whatwg.org/multipage/media.html#dom-media-videotracks
     JS::GCPtr<VideoTrackList> m_video_tracks;
 
+    // https://html.spec.whatwg.org/multipage/media.html#dom-media-texttracks
+    JS::GCPtr<TextTrackList> m_text_tracks;
```

Comment:
- Who sets this to a non-null value?
- Who makes sure this is marked for GC?

Reply:
:facepalm: Oops, I didn't stage those two changes.

Resolved in https://github.com/LadybirdBrowser/ladybird/pull/447/commits/4bd93c454d63b863e8adbcc582e61525a8535dbe  :)